charts/rancher-vsphere-cpi/templates/_helpers.tpl

```diff
@@ -35,3 +35,18 @@
 {{- end -}}
 {{- join ", " $versions -}}
 {{- end -}}
+
+{{/*
+Windows cluster will add default taint for linux nodes,
+add below linux tolerations to workloads could be scheduled to those linux nodes
+*/}}
+{{- define "linux-node-tolerations" -}}
+- key: "cattle.io/os"
+  value: "linux"
+  effect: "NoSchedule"
+  operator: "Equal"
+{{- end -}}
+
+{{- define "linux-node-selector" -}}
+kubernetes.io/os: linux
+{{- end -}}
```

Comment:
Will the lack of a newline mess up the helper function/template?

Reply:
it won't make a difference